corehq/apps/analytics/templates/analytics/analytics_js.html

```diff
@@ -12,5 +12,6 @@
   <script src="{% static 'analytix/js/hubspot.js' %}"></script>
   <script src="{% static 'analytix/js/drift.js' %}"></script>
   <script src="{% static 'analytix/js/appcues.js' %}"></script>
-  <script src="{% static 'analytix/js/gtm.js' %}"></script>
+  {# gtx is the filename because some adblockers regex *gtm* #}
```

Comment:
Not sure it matters much, but how did you determine that "some adblockers regex `*gtm*`"? That seems like a potentially common sequence of characters, and I think it would block many other scripts as well.

It may not matter for this PR since you have confirmed that the rename does indeed avoid common ad blockers.

Reply:
My bad. I don't think this comment is factually correct. 
It is hard to say without elaborate testing with various ad blockers how they detect analytics scripts. Although for `gtm` it was only blocked by ublock-origin , and can confirm that it does not regex for *gtm*.

Going to update this comment.